src/telemetryReporter.ts

```diff
@@ -111,4 +124,22 @@ export default class TelemetryReporter extends vscode.Disposable {
         });
 
     }
+
+    private format(args: any): string {
```

Comment:
This method seems overly complicated given that you always just pass in `[string, object, string]` params.

Reply:
It was copied over from what we have in core as I wanted to maintain the same format. But I agree, we can simplify